blockchain/chain_sync/src/network_context.rs

```diff
@@ -172,35 +175,69 @@ where
         let global_pre_time = SystemTime::now();
         let bs_res = match peer_id {
```

Comment:
naming: is bs for bootstrap? can we have an expanded name for this?

Reply:
Maybe it stands for for `BlockStore`?

Reply:
I will abstain from noting what `bs` usually means. :smile: That's why I'm for full names, even if it gets very verbose. We all have autocompletion anyway; it's not the 90s.

Reply:
I will abstain from noting what `bs` usually means. :smile: That's why I'm for full names, even if it gets very verbose. We all have autocompletion anyway; it's not the 90s.

Reply:
According to the logic, this comes from p2p network, I'm not sure if `bs` stands for `blockstore` here : )